accelerators-home/modules/ROOT/pages/simplifying-assets.adoc

```diff
@@ -0,0 +1,70 @@
+= Considerations for Simplifying Accelerator Assets
+
+MuleSoft Accelerators follow an API led approach to connectivity, which separates app, process, and system integration layers while enabling a high level of reusability, particularly with System APIs. The low maintenance costs and ability to enable future growth make this process highly recommended. However, if you want a less complex approach, follow this guidance to simplify your integrations and architecture. 
+
+== Simplify Accelerator Assets
+
+A consistent driving factor behind the design and implementation of use cases supported by the MuleSoft Accelerators has always been to keep the assets and corresponding solutions as simple as possible while still providing a solid, functional, and maintainable architecture for customers to build upon. This has been achieved through the following key solution design principles:
+
+* Follow the API-led connectivity approach but create APIs only where necessary to support clean integration with external apps and systems.
+* Keep exception handling clean and simple, allowing for straightforward extension and customization. 
+* Avoid the use of heavy logging, monitoring, or exception handling frameworks.
+* Leverage a reusable canonical data model within the business process layer to simplify communication between layers.
+* Use messaging only where deemed necessary to support high-volume deployments.
+* Use inline sub-flows instead of top-level flows wherever possible.
+
+The assets being developed for the MuleSoft Accelerators were also intended to be flexible and extensible, supporting not only the use cases that have been solutioned but also other use cases needed by customers to support their business. Additional endpoints have been included in some APIs, especially System APIs shared across multiple use cases, which may not be required in all customer deployments. The reference architectures for most solutions have also been designed for long-term growth and evolution, which sometimes results in additional apps and interactions that are not always required for smaller deployments. 
+
+The following sections describe steps you can take to reduce the number of flows in accelerator apps.
+
+=== Remove Unused APIkit Handlers
+
+Remove scaffolded flows from the main router flow file in APIkit, which is usually the `*-api.xml` file. Leave the API specification unchanged instead of rescaffolding, because you always edit the file manually to remove the scaffolded flows. Avoid circumventing the APIkit router with a plain choice router, because that makes the router less discoverable and increases maintenance costs.
```

Comment:
Consider making this a bullet list for scannability. Each sentence is a nice imperative.